ctx_test.go

```diff
@@ -164,11 +174,21 @@ func TestContext_QueryParam(t *testing.T) {
 
 		paramBool, err = c.QueryParamBoolErr("notfound")
 		require.Error(t, err)
-		require.False(t, paramBool)
+		assert.Zero(t, paramBool)
+		notFoundErr := &internal.QueryParamNotFoundError{}
+		require.ErrorAs(t, err, notFoundErr)
+		assert.Equal(t, "param notfound not found", notFoundErr.Error())
+		assert.Equal(t, http.StatusUnprocessableEntity, notFoundErr.StatusCode())
+		assert.Equal(t, "param notfound not found", notFoundErr.DetailMsg())
 
 		paramBool, err = c.QueryParamBoolErr("other")
 		require.Error(t, err)
-		require.False(t, paramBool)
+		require.Zero(t, paramBool)
```

Comment:
I'm surprised by this change.

Was it suggested by testifylint?

Reply:
let me check